templates/index.html.twig

```diff
@@ -67,6 +67,14 @@
                                         </a>
                                     </span>
                                 {% endif %}
+
+                                {% if app.user %}
+                                    <span class="m-2 d-inline-block invisible" data-toggle="appear" data-class="animated fadeInUp" data-timeout="600">
+                                        <a class="btn btn-success px-4 py-2" data-toggle="click-ripple" href='{{ path('fos_user_profile_show') }}'>
+                                            <i class="fa fa-fw fa-shopping-cart mr-1"></i> Profile
+                                        </a>
+                                    </span>
+                                {% endif %}
                             </div>

```

Comment:
You could add two buttons to access to movie and tv list even if use is not connected

Reply:
Yes Sir. Like i said in the pull request, routes haven't been implemented